great_expectations/execution_engine/sparkdf_execution_engine.py

```diff
@@ -480,6 +491,31 @@ def get_domain_records(
 
         return data
 
+    def _combine_row_conditions(
+        self, row_conditions: List[RowCondition]
+    ) -> RowCondition:
+        """Combine row conditions using AND if type_ is SPARK_SQL
+
+        Args:
+            row_conditions: Row conditions of type Spark
+
+        Returns:
+            Single Row Condition combined
+        """
+        assert all(
+            [
+                condition.type_ == RowConditionParserType.SPARK_SQL
+                for condition in row_conditions
+            ]
+        ), "All row conditions must have type SPARK_SQL"
+        raw_conditions: List[str] = [
+            row_condition.condition for row_condition in row_conditions
+        ]
+        joined_condition: str = " AND ".join(raw_conditions)
```

Comment:
@anthonyburdi If possible, could we use the `SQLAlchemy` way?  We have a few examples; happy to discuss.  Thanks.